scripts/build-wvr-ud.js

```diff
@@ -145,7 +144,12 @@ concat(stlyeFiles, `${WVR_UD_STATIC_ASSETS_DIR}/styles.css`).finally(() => {
     if(fs.lstatSync(sa).isDirectory()) {
       copyFolderSync(sa, `${CONFIG.output}/${fileName}`);
     } else {
-      fs.copyFileSync(sa, `${CONFIG.output}/${fileName}`);
+      if( (sa.indexOf('static-assets/wud.js') > -1) && (fs.existsSync(`${NM_WVR_UD_DIR}/static-assets/wud.js`)) ) {
```

Comment:
This will only copy the file from node modules, if it exists in the local .wud dir. What if they don;t have a wud.js in their .wud dir? in that instance we would never end up in this block, and the node_module version would not get copied.

Reply:
In addition, if they do have a local copy of wud.js, it should be used instead of the wud.js from node_modules

Reply:
Modified the condition